test/junit/scala/tools/nsc/interpreter/CompletionTest.scala

```diff
@@ -180,38 +150,6 @@ class CompletionTest {
     checkExact(completer, "1 + 1.toCha")("toChar")
   }
 
-  @Test
-  def defString(): Unit = {
-    val completer = setup()
-
-    // Double Tab on a fully typed selection shows the def string
```

Comment:
This now happens on a single tab, right? could we keep the tests?

Reply:
The tests need to be redone since the code has changed. This was a corner I cut late last night; I've set a personal todo to bring testing back, but I think it can wait until after 2.13.2.